charts/hono/values.yaml

```diff
@@ -1257,6 +1261,18 @@ deviceRegistryExample:
     # configuration property of the component's readiness probe.
     # The value of the top level "readinessProbeInitialDelaySeconds" property will be used if not set.
     readinessProbeInitialDelaySeconds:
+    # resources contains the container's requests and limits for CPU and memory
+    # as defined by the Kubernetes API.
+    # If not specified here, then the values from "deviceRegistryExample.resources" are used.
+    # Refer to https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+    # for a description of the properties' semantics.
+    resources:
+      requests:
+        cpu: "200m"
+        memory: "400Mi"
```

Comment:
Isn't 300Mi (as for the protocol adapters) enough here? `kubectl top pod` showed 263Mi for the pod initially when I tested this.
Just saying, because with 400Mi here, we have 2072Mi for the complete standard deployment already.

Reply:
With 300Mi the integration tests of Hono when run with the embedded H2 DB sometimes fail, i.e. the registry container crashes ...

Reply:
Oh, ok.